recipes/eleuther_eval.py

```diff
@@ -163,101 +440,84 @@ class EleutherEvalRecipe(EvalRecipeInterface):
 
     Features:
         - Single GPU evaluation. Multi-GPU evaluation is currently not supported.
-        - Loading model in fp32 or bf16. Fp16 is currently not supported.
-
-    We recommend launching evaluation using the tune CLI:
+        - Quantization and torch.compile (for text-only models) is supported.
```

Comment:
Are you planning to add in compile support?

Reply:
Yes definitely.

Reply:
Take out compile for now?